exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -81,7 +85,7 @@ void createTestKitForSingleService(TestKit testKit) {
 
   @Test
   void createTestKitWithBuilderForSingleService() {
```

Comment:
It is probably no longer needed, isn't it?

Reply:
But why? I think it's still a viable test.

Reply:
Sorry, wrong method — the question is for createTestKitForSingleService^.